backend/app/src/main/java/com/moh/phlat/backend/service/dto/UiColumnName.java

```diff
@@ -0,0 +1,11 @@
+package com.moh.phlat.backend.service.dto;
+
+import lombok.Builder;
+import lombok.Getter;
```

Comment:
don't we need getter and setter for this? If not, these are unused imports

Reply:
Unused imports have been checked and removed.